server/schemas/userSchema.js

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+
+export default z.object({
```

Comment:
Instead of creating a new schemas folder and file, what do you think about including this in the models/User.js file...? The schema could be a static field in the class...? Then the usage could be like `User.schema.parse(data)`...

Reply:
Moved into User object!